### PR TITLE
HW4: Task 4

### PR DESCRIPTION
**Pull request by ValeriL**
1. Again #noqa because of Cognitive Complexity (6 > 5)
2. The name of the file starts with 'test' for checking via GH actions.

### CODE REVIEW

File: Homework/homework4/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Instructions: How to run a doctest
+
+1. Install Python 3.8 (https://www.python.org/downloads/)
+2. Install pytest `pip install pytest`
+3. Clone the repository https://github.com/ValeriL/Epam_Python \
+    `git clone https://github.com/ValeriL/Epam_Python`
+4. Checkout branch hw4_4 \
+    `git checkout hw4_4`
+6. Run a doctest for test_task4.py \
+    `pytest Homework/homework4/test_task4.py`
```
Comment (Cjkjvfnby) [inline Homework/homework4/README.md:10]: Pytest does not run doctest by default.  Why does it work for you?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]
> The name of the file starts with 'test' for checking via GH actions.

Is it really required for doctest?

File: Homework/homework4/README.md (hunk 1 above)
Comment (ValeriL) [reply Homework/homework4/README.md:10]: So the point is to make pytest see doctests in this file and check it without straight mention "test" in the name of the file?

Comment (Cjkjvfnby) [reply Homework/homework4/README.md:10]: Create a folder outside this repository. Copy your file there and run pytest.

File: Homework/homework4/task4.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        elif i % 3 == 0:
+            fizzbuzz_numbers.append("fizz")
+        elif i % 5 == 0:
+            fizzbuzz_numbers.append("buzz")
+        else:
+            fizzbuzz_numbers.append(str(i))
+    return fizzbuzz_numbers
+
+
+if __name__ == "__main__":
```
Comment (Cjkjvfnby) [inline Homework/homework4/task4.py:62]: Remove this.  This is not required.

**Review by Cjkjvfnby — APPROVED** [review]
I'll describe the solution I have expected at the meeting.

**Pull request merged**